### PR TITLE
Add validation to check target column in query instance

### PR DESCRIPTION
**Pull request by gaugup**
- Alerting users when they look to generate counterfactuals that they don't have target in their query instances.
-  
Signed-off-by: gaugup <gaugup@microsoft.com>

### CODE REVIEW

**Comment by gaugup** [comment]
@soundarya98 could you take a look at this PR?

Regards

File: tests/conftest.py [hunk 1]
```diff
@@ -111,11 +119,11 @@ def sample_custom_query_5():
 
 
 @pytest.fixture
-def sample_custom_query_4():
+def sample_custom_query_6():
```
Comment (soundarya98) [inline tests/conftest.py:122]: Probably something for later, but we could consider renaming the queries according to their functions

File: dice_ml/explainer_interfaces/explainer_base.py [hunk 2]
```diff
@@ -156,6 +158,10 @@ def setup(self, features_to_vary, permitted_range, query_instance, feature_weigh
 
     def check_query_instance_validity(self, features_to_vary, permitted_range, query_instance, feature_ranges_orig):
         for feature in query_instance:
+            print(feature)
```
Comment (soundarya98) [inline dice_ml/explainer_interfaces/explainer_base.py:161]: We can remove the print statement

File: dice_ml/explainer_interfaces/explainer_base.py [hunk 3]
```diff
@@ -139,6 +139,8 @@ def _generate_counterfactuals(self, query_instance, total_CFs,
         pass
 
     def setup(self, features_to_vary, permitted_range, query_instance, feature_weights):
+        # import pdb
```
Comment (soundarya98) [inline dice_ml/explainer_interfaces/explainer_base.py:142]: Can remove these commented lines too

**Comment by soundarya98** [comment]
> @soundarya98 could you take a look at this PR?
> 
> Regards

Thanks for the pull request! I've added my comments, other than that, the PR looks good!

File: dice_ml/explainer_interfaces/explainer_base.py (hunk 3 above)
Comment (gaugup) [reply dice_ml/explainer_interfaces/explainer_base.py]: Done

File: dice_ml/explainer_interfaces/explainer_base.py (hunk 2 above)
Comment (gaugup) [reply dice_ml/explainer_interfaces/explainer_base.py]: Done

File: tests/conftest.py (hunk 1 above)
Comment (gaugup) [reply tests/conftest.py:122]: Thanks the suggestion. Yeah, maybe addressable at a later time.

**Review by amit-sharma — APPROVED** [review]

**Pull request merged**